essentials/src/types.rs

```diff
@@ -35,14 +41,25 @@ pub type SubxtCall = runtime::RuntimeCall;
 
 pub type ClaimQueue = BTreeMap<u32, VecDeque<Option<ParasEntry>>>;
 
+// TODO: Take it from runtime types v5
+/// How a free core is scheduled to be assigned.
+pub struct CoreAssignment {
```

Comment:
Just a heads up, on master, this type has changed a bit. See here: https://github.com/paritytech/polkadot/blob/759fe213acd6071b4a2bfebcef08be8a5f2ecc15/runtime/parachains/src/scheduler/common.rs#L79

Reply:
Thank you. Here it's ok, we need it to support "old", not updated to v5 blockchains.